packages/stencil/components/button/src/components/osds-button/styles/osds-button.size.scss

```diff
@@ -20,4 +20,12 @@
       padding: ods-get-size-property($sizes, padding-y) ods-get-size-property($sizes, padding-x);
     }
   }
+
+  /** circle */
+  [circle] {
+    @include osds-button-on-selected-host() {
+      padding: 4px; height: auto; line-height: 0.4em; border-radius: 50%;
```

Comment:
probably easier to read if we keep every prop on a separate line
(a scss linter would be nice to add)

Reply:
ok